subprojects/cfa/cfa/README.md

```diff
@@ -39,18 +39,31 @@ Variables of the CFA can have the following types.
 * `rat`: Rational numbers (implemented as SMT reals).
 * `[K] -> V`: SMT arrays (associative maps) from a given key type `K` to a value type `V`.
 * `bv[L]`: Bitvector of given length `L`. _This is an experimental feature. See the [details](doc/bitvectors.md) for more information._
+* `fp[E:S]`: Floating point of given size exponent `E` and significand `S`. Significand size should include the hidden bit as well.` 
```

Comment:
Can you maybe put a reference here to any standards that this type follows? And it is also worth mentioning that this type corresponds to the SMT LIB theory of floats.